components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1534,6 +1540,8 @@
 <!-- viscosity  -->
 
 <nu_top> 2.5e5 </nu_top>
+<nu_top dyn_target="preqx"   hgrid="ne45np4"> 2.5e5 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne45np4"> 2.5e5 </nu_top>
```

Comment:
these two lines are not needed, since they are using the default value?

Reply:
I originally had different values, I'll remove these.